.gitignore

```diff
@@ -4,5 +4,5 @@ Cargo.lock
 /micro-rdk-server/viam.json
 /canary/.venv/*
 .vscode
-components_esp32.lock
+components_esp32*.lock
```

Comment:
I wonder if we are supposed to be checking these files in, much like we do with `Cargo.lock`?